### PR TITLE
Docs: add feature toggle wording guidance

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: docs/sources/contribute-documentation/contribute-release-notes/index.md [hunk 1]
```diff
@@ -344,3 +345,6 @@ Follow these guidelines to ensure that your What's new or release notes content
   Now, the state doesn't update."
 
 - **For changes or updates to features, provide brief descriptions.**
+
+
+  <p></p>
```
Comment (imatwawana) [inline docs/sources/contribute-documentation/contribute-release-notes/index.md:350]:
```suggestion
```

**Review by brendamuir — APPROVED** [review]

File: docs/sources/contribute-documentation/contribute-release-notes/index.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ When you’re ready to add a What’s new entry, complete the following steps:
            <p>If you want to view some best practices around what to write here, refer to <a href="#writing-guidelines-for-whats-new-content">Writing guidelines for What’s new</a>.</p>
            <p>Add any images and a link to your public YouTube video here.</p>
            <p>If you need more information on adding an image, refer to <a href="https://grafana.com/docs/writers-toolkit/write/image-guidelines/">Image, diagram, and screenshot guidelines</a>.</p>
+           <p>If you need to mention a feature flag, use this format: To try out Trace to profiles, enable the <code>traceToProfile</code> <a href="https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/">feature toggle</a>.</p>
```
Comment (JStickler) [inline docs/sources/contribute-documentation/contribute-release-notes/index.md:118]: Would it be helpful to also include how to achieve the monospace formatting?  I don't actually know what the best practices are for formatting content written in the CMS.

Comment (imatwawana) [reply docs/sources/contribute-documentation/contribute-release-notes/index.md:118]: The CMS allows you to use Markdown, so folks shouldn't need an explanation. They've already been doing this anyway.

**Review by jdbaldry — APPROVED** [review]
LGTM

**Pull request merged**